### ISSUE TITLE
Lighthouse's over-zealous error reporting

### ISSUE THREAD

**Issue by trevorical** [GoogleChrome/lighthouse#11930]
Having now used Lighthouse to audit a number of websites, I remain irked by a number of points where Lighthouse returns an error when there is no real error present.
The worst is the error message returned when the < li > elements of an < ul > or < o l> are set up as links. By this I mean that the whole < li > is a link, not just the text in it.  Lighthouse flags this as an error on the pretext that an < ol > or an < ul > cannot include anything other than < li > elements.  But to make a whole < li > element operate as a link, as in < ul > formatted menu tabs, the elements have to be structured in the order < ul >< a >< li >content< /li >< /a >< /ul >.
  I can see no intrinsic reason for this to be marked as an error. Please update Lighthouse to accept this which is user-friendly on all devices.

Another point in brief: Lighthouse's requirement of  color-contrast ratio of at least 4.5:1 really needs to be softened.  For instance, it rejects highly visible contrasts such as yellow text on red background (FFFF00 on FF0000), which stands out on any web page.
  The color contrast requirement also plays havoc with any "sophisticated" use of matching colors, renders some logo colors unusable outside of the logo without incurring a Lighthouse penalty, and fails to take account of the fact that there are occasions when a website needs to have less eye-catching contrast between text and background. Such as including a discreet contact link that will not be immediately noticed on by scammers and spammers

**Comment by patrickhulce** [comment]
Thanks for filing @trevorical! It sounds like you're unhappy with the audits for accessibility best practices, is that correct?

The rationale and explanation for each of those can be found in the documentation links that accompany each audit which you might find helpful. The implementation for these comes from [a community of accessibility experts](https://github.com/dequelabs/axe-core) and Lighthouse will not be creating one-off exceptions based solely on developer implementation frustrations. If there's a case you believe is actually in the best interest of an accessible site that isn't being met by the current best practices, an issue in the [axe-core repo](https://github.com/dequelabs/axe-core) for discussion would be the best place to discuss a better solution.

**Comment by trevorical** [comment]
No, generally speaking I'm very happy with the Lighthouse audits, which are pragmatic and generally clear. I have indeed checked out the documentation for the issues I have raised, but it does not address the _specific_ points I have raised; and I do not for one moment imagine that Lighthouse would or even could create one-off exceptions. But yes, I do believe that the issues I have highlighted are not duly taken into consideration by Lighthouse, and that they are areas in which Lighthouse could be improved
   My observations come from someone who is solely a webmaster, not an IT developer, so I see how Lighthouse works very much as an end user (I never studied IT, I just grew up with it, starting in the days before Google had even been invented). A tool like Lighthouse needs feedback from end-users who will view the product in a less abstract/theoretical way than developers..... like "yellow text on a red background is highly visible" (which is the practical take on the question, and explains why this color contrast is used worldwide on danger / alarm notices) rather than "#FFFF00 on#FF000 has a contrast ratio of less than 4.5:1 so it can't be visible enough." which is the abstract / theoretical take on it.,
   I followed your link to axe-core repo, but don't see how to start a new discussion there as there is no "new issue" button. So if anyone reading this would like to escalate the points raised so that they are brought to the attention of the people who might act on them, that would be great.
